Pico/mainRP.ino

```diff
@@ -0,0 +1,97 @@
+#include <Arduino.h>
+#include "Motors/Motors.h"
+#include <typeinfo>
+
+#define PIN_SERIAL1_TX (0u)
+#define PIN_SERIAL1_RX (1u)
+
+uint8_t motor4In1 = 25;
+uint8_t motor4In2 = 6;
+uint8_t motor4PWM = 20;
+
+uint8_t motor3In1 = 14;
+uint8_t motor3In2 = 11;
+uint8_t motor3PWM = 21;
+
+uint8_t motorIn1 = 8;
+uint8_t motorIn2 = 9;
+uint8_t motorPWM = 10;
+
+uint8_t motor2In1 = 22;
+uint8_t motor2In2 = 23;
+uint8_t motor2PWM = 15;
```

Comment:
The information of the PINS could be added to a `Constants.h` file with an static class for easier data handling.

Reply:
Great! This will also be fixed while I am away on a new issue